src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -397,7 +397,9 @@ export class PostgresCohortMembersService {
           return APIResponse.error(
             res,
             apiId,
-            API_RESPONSES.USER_NOTFOUND,
+            whereClause['cohortId']
```

ESLint

[error] Replace `'cohortId'` with `"cohortId"`

(prettier/prettier)